src/components/list/row.tsx

```diff
@@ -22,13 +23,13 @@ export const Row = (props: PropsWithChildren<Props>) => {
       <span className="truncate">{props.name}</span>
       <span className={`flex flex-initial flex-shrink-0 flex-row items-center gap-2`}>
         {props.children}
-        <a
+        <Link
```

Comment:
hmm next Link should not be used for external links

Reply:
Shouldn't all rows be internal links anyway?

Reply:
no sometimes we said these would link to an external website

Reply:
Hmm that's right, we'll need to adjust this then

Reply:
i guess thats not MVP so we can change this when that happens

Reply:
Agreed